addons/account/migrations/13.0.1.1/post-migration.py

```diff
@@ -238,24 +238,46 @@ def migration_invoice_moves(env):
             AND ((ail.product_id IS NULL AND aml.product_id IS NULL) OR ail.product_id = aml.product_id)
             AND ((ail.uom_id IS NULL AND aml.product_uom_id IS NULL) OR ail.uom_id = aml.product_uom_id)
         """
-    # Try first with a stricter criteria for matching invoice lines with account move lines
-    openupgrade.logged_query(
-        env.cr, query + minimal_where + """
-            AND ail.account_id = aml.account_id
-            AND ai.commercial_partner_id = aml.partner_id
-            AND ((ail.account_analytic_id IS NULL AND aml.analytic_account_id IS NULL)
-                OR ail.account_analytic_id = aml.analytic_account_id)
-        RETURNING aml.id""",
-    )
-    aml_ids = tuple(x[0] for x in env.cr.fetchall())
-    # Try now with a more relaxed criteria, as it's possible that users change some data on amls
-    openupgrade.logged_query(
-        env.cr, query + minimal_where + """
-            AND aml.old_invoice_line_id IS NULL
-            AND rc.anglo_saxon_accounting IS DISTINCT FROM TRUE
-        RETURNING aml.id""",
-    )
-    aml_ids += tuple(x[0] for x in env.cr.fetchall())
+    aml_ids = tuple()
```

Comment:
I don't think we need this variable, as it's only used in one `if` in:

https://github.com/OCA/OpenUpgrade/blob/8d6b578ba656d39c407bcf803f8251b1f2030044/addons/account/migrations/13.0.1.1/post-migration.py#L345

and I think we can run that query nevertheless it, and if not, we can just have a flag for activating it.

Reply:
but we need it because inside the while we are using `aml_ids +=`

Reply:
I don't see its usage. Please check again.

Reply:
lines 256, 266 and 280

Reply:
That aren't read operations, but write ones, and thus, useless. We can remove all. That's my point.

Reply:
ok, done